lib/builtin.t

```diff
@@ -67,6 +67,18 @@ package FetchStoreCounter {
     weaken($ref);
     undef $arr;
     is($ref, undef, 'ref is now undef after arr is cleared');
+
```

Comment:
Plus if we're going to document that `undef` is permitted, it probably want a test here to check it doesn't throw